spec/features/authentication/log_in_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.feature "LogIns", type: :feature do
```

Comment:
Not sure if `type: :feature` is necessary in case of using `RSpec.feature`, I only used it alongside with `RSpec.describe`. So please check if it can be omitted.